.travis.yml

```diff
@@ -1,15 +1,21 @@
 language: ruby
 cache: bundler
 sudo: false
-
 rvm:
-  - 2.1.8
-  - 2.2.4
-  - 2.3.0
+- 2.1.8
+- 2.2.4
+- 2.3.0
+- 2.4.2
 script: bundle exec rspec
-
 before_install:
-  - mkdir travis-phantomjs
-  - wget https://github.com/JordiPolo/phantomjs/blob/master/phantomjs-2.1.1-linux-x86_64.tar.bz2?raw=true -O $PWD/travis-phantomjs/phantomjs-2.1.1-linux-x86_64.tar.bz2
-  - tar -xvf $PWD/travis-phantomjs/phantomjs-2.1.1-linux-x86_64.tar.bz2 -C $PWD/travis-phantomjs
-  - export PATH=$PWD/travis-phantomjs/phantomjs-2.1.1-linux-x86_64/bin:$PATH
+- mkdir travis-phantomjs
```

Comment:
why the indent change? it makes the file much harder to read

Reply:
The Travis tool sets up the encrypted key and formats the file this way